base/server/python/pki/server/subsystem.py

```diff
@@ -224,8 +227,8 @@ def validate_system_cert(self, cert_id=None):
             print(subprocess.check_output(
                 cmd,
                 stderr=subprocess.STDOUT))
-        except subprocess.CalledProcessError as e:
-            print("pki-server subsystem-cert-validate stdout output:\n", e.output)
+        except subprocess.CalledProcessError as v:
+            print("pki-server subsystem-cert-validate stdout output:\n", v.output)
```

Comment:
Sorry I missed your question. The original code combines the stdout and stderr. If you want to separate them out you probably could do this (I haven't tried it myself):
```
result = subprocess.run(cmd, capture_output=True, text=True)
print(result.stdout.strip('\n'))
if result.returncode == 0:
    print(result.stderr.strip('\n'))
```
https://stackoverflow.com/questions/41171791/how-to-suppress-or-capture-the-output-of-subprocess-run